tests/test_import.py

```diff
@@ -292,3 +292,71 @@ def test_all_zero_donations(self, user, testapp):
         assert Batch.query.count() == existing_batches
         # We don't want the data repair form to show up
         assert "Řádky s chybami" not in res
+
+    @pytest.mark.parametrize(
+        ("input_line", "expected_first_name", "expected_last_name"),
+        (
+            (
+                "205225295;TOMÁŠ;VESELÁ,dr;HAMERSKÁ 810;VELKÉ HAMRY;31777;985;69",
+                "Dr. TOMÁŠ",
+                "VESELÁ",
+            ),
+            (
+                "0352152680;LUBOMÍR;ŠIMEK,BCA.;HUSOVA 2;IVANOVICE NA HANÉ;69710;409;1",
+                "BcA. LUBOMÍR",
+                "ŠIMEK",
+            ),
+            (
+                "1860231538;VÁCLAV;MAREŠOVÁ    rndr;RYCHVALDSKÁ 11;RÝMAŘOV;11252;180;6",
+                "RNDr. VÁCLAV",
+                "MAREŠOVÁ",
+            ),
+            (
+                "431229128;MARTIN;DVOŘÁK ing.MGR.;SKALICE 451;MILETÍN;99473;515;3",
+                "Mgr. Ing. MARTIN",
```

Comment:
Tady není dodrženo toto: [https://prirucka.ujc.cas.cz/?id=783&dotaz=titul](https://prirucka.ujc.cas.cz/?id=783&dotaz=titul) (je tam sice napsáno, že oficiální pravidlo na to není, záleží jak moc je požadováno dodržení tohoto pravidla).
V databázi by měly být tyto kombinace dvou titulů před jménem:
* `Ing.Phdr.`
* `Ing.,PHDr.`